mayavi/tools/helper_functions.py

```diff
@@ -1179,3 +1181,10 @@ def test_triangular_mesh():
     t = np.r_[0, t]
 
     return triangular_mesh(x, y, z, triangles, scalars=t)
+############################################################
```

Comment:
Please remove these unnecessary commented lines.  Also no need to add an import for gcf, can't you just use `tools.gcf`?